src/parcsr_mv/par_csr_matrix.c

```diff
@@ -227,8 +249,8 @@ hypre_ParCSRMatrixSetNumNonzeros( hypre_ParCSRMatrix *matrix )
    hypre_CSRMatrix *offd;
    HYPRE_Int *offd_i;
    HYPRE_Int local_num_rows;
-   HYPRE_Int total_num_nonzeros;
-   HYPRE_Int local_num_nonzeros;
+   HYPRE_BigInt total_num_nonzeros;
```

Comment:
maybe total_num_nonzeros should be of type long int anyway. a matrix <2B can have  nnz >2B. but this is not a big prob